parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -55,6 +55,10 @@ private ParquetUtil() {}
 
   private static final long UNIX_EPOCH_JULIAN = 2_440_588L;
 
+  // Column index configuration constants (shared with ParquetWriter)
+  public static final String COLUMN_INDEX_TRUNCATE_LENGTH = "parquet.columnindex.truncate.length";
+  public static final int DEFAULT_COLUMN_INDEX_TRUNCATE_LENGTH = 64;
```

Comment:
Instead defining these again, could we use `ParquetProperties.DEFAULT_COLUMN_INDEX_TRUNCATE_LENGTH` and `ParquetOutputFormat.COLUMN_INDEX_TRUNCATE_LENGTH`?

Reply:
Good call.